apps/site/pages/components/organisms/filter.mdx

```diff
@@ -0,0 +1,476 @@
+---
+description: The Filter component is used to filter products inside the Product Listing Page (PLP) and Search Page.
+keywords:
+  - UI library
+  - ecommerce
+  - react
+sidebar_custom_props:
+  image: https://vtexhelp.vtexassets.com/assets/docs/src/PaymentMethods___b0f344b254360daa53406cd631887adf.png
+---
+
+import { Tab, Tabs } from 'nextra-theme-docs'
+import PropsSection from 'site/components/PropsSection'
+import { TokenTable, TokenRow, TokenDivider } from 'site/components/Tokens'
+import { OverviewSection } from 'site/components/OverviewSection'
+import { SectionItem, SectionList } from 'site/components/SectionItem'
+import FilterUsage from 'site/components/Filter/FilterUsage'
+import { Button, Toast, useUI, UIProvider, Filter } from '@faststore/ui'
+import { SearchProvider } from '@faststore/sdk'
+import path from 'path'
+import { useSSG } from 'nextra/ssg'
+import { getComponentPropsFrom } from 'site/components/utilities/propsSection'
+
+export const getStaticProps = () => {
+  const componentPath = path.resolve(__filename)
+  const components = [
+    'Filter.tsx',
+    'Facets.tsx',
+    'FacetBooleanItem.tsx',
+    'FacetRange.tsx',
+  ]
+  const [filterProps, facetsProps, facetBooleanItemProps, facetRangeProps] =
+    getComponentPropsFrom(componentPath, components)
+  return {
+    props: {
+      // We add an `ssg` field to the page props,
+      // which will be provided to the Nextra `useSSG` hook.
+      ssg: {
+        filterProps,
+        facetsProps,
+        facetBooleanItemProps,
+        facetRangeProps,
+      },
+    },
+  }
+}
+
+export const FilterPropsSection = ({ component }) => {
+  // Get the data from SSG, and render it as a component.
+  const { filterProps, facetsProps, facetBooleanItemProps, facetRangeProps } = useSSG()
+  return {
+    Filter: <PropsSection propsList={filterProps} />,
+    Facets: <PropsSection propsList={facetsProps} />,
+    FacetBooleanItem: <PropsSection propsList={facetBooleanItemProps} />,
+    FacetRange: <PropsSection propsList={facetRangeProps} />,
+  }[component]
+}
+
+<header>
+
+# Filter
+
+The Filter component is used to filter products inside the Product Listing Page (PLP) and Search Page.
+
+</header>
+
```

Comment:
The `Filter` component is a compound of the following:

Reply:
Does adding this sentence okay?

Reply:
Sure! I'll add this.